examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
Thanks @ahashibon, my main concern is serialization into HDF5. It is preferable in general that DataContainers are not contain variable length components. However, the configuration containers do not have to be the same DataContainers  object as the one used for Particles or Mesh Elements (see issue https://github.com/simphony/simphony-common/issues/29). But I think that the work you guys are doing will help find the more appropriate solution based on these usecases.